custom_components/opensprinkler/__init__.py

```diff
@@ -361,6 +375,13 @@ async def stop(self):
 
 
 class OpenSprinklerProgramEntity:
+    @property
+    def available(self):
```

Comment:
This should ultimately call super I think. Meaning, fall back to the coordinator value.